specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2024-03-01-preview/purviewdatamap.json

```diff
@@ -0,0 +1,8489 @@
+            "schema": {
+              "$ref": "#/definitions/SuggestResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/AtlasErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Discovery_Suggest": {
+            "$ref": "./examples/Discovery_Suggest.json"
+          }
+        }
+      }
+    },
+    "/navigate": {
+      "post": {
+        "tags": [
+          "Discovery"
+        ],
+        "description": "Navigate entities by itemPath.",
+        "operationId": "Discovery_Navigation",
+        "x-ms-examples": {
+          "Step 1 Discovery Navigate from root": {
+            "$ref": "./examples/Discovery_Navigate_01.json"
+          },
+          "Step 2 Discovery Navigate from assetType": {
+            "$ref": "./examples/Discovery_Navigate_02.json"
+          },
+          "Step 3 Discovery Navigate from server": {
+            "$ref": "./examples/Discovery_Navigate_03.json"
+          },
+          "Step 4 Discovery Navigate from databases category": {
+            "$ref": "./examples/Discovery_Navigate_04.json"
+          },
+          "Step 5 Discovery Navigate from database": {
+            "$ref": "./examples/Discovery_Navigate_05.json"
+          },
+          "Step 6 Discovery Navigate from schemas category": {
+            "$ref": "./examples/Discovery_Navigate_06.json"
+          },
+          "Step 7 Discovery Navigate from schema": {
+            "$ref": "./examples/Discovery_Navigate_07.json"
+          },
+          "Step 8 Discovery Navigate from tables category": {
+            "$ref": "./examples/Discovery_Navigate_08.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "NavigationRequest",
+            "description": "An object specifying the navigation criteria.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NavigationRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/maxpagesize"
+          },
+          {
+            "$ref": "#/parameters/continuationToken"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. On successful get the entities.",
+            "schema": {
+              "$ref": "#/definitions/NavigationResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
Why only this API uses `ErrorResponse`?

I believe it is not because it is the only non-atlas API. The `search` API above is not atlas.

Reply:
The reason we use ErrorResponse in the Navigate API is to ensure that our responses align with the best practices of the Azure REST API. This API is not dependent on Atlas. Regarding the Search API, I can share this information with the team and see if we can switch to using ErrorResponse as well. This may require some code changes.

Reply:
The consistency of the service is of higher priority.

It would be confusing for user, if only 1 API take a different schema of error response. Please double check with team.